lib/rules/web/ocm_console/ocm_workflow.xyaml

```diff
@@ -13,36 +13,36 @@ create_osd_cluster:
     action: overview_tab_loaded
 archive_cluster_from_list:
```

Comment:
seems like `archive_cluster_from_cluster_list_page` is more readable.

Reply:
So do the left.

Reply:
So do the left.  Especially the output steps, sometimes we can not save the words.